webextension/js/popup.js

```diff
@@ -857,8 +881,10 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     const userContextId = Logic.currentUserContextId();
     const assignments = await Logic.getAssignmentObjectByContainer(userContextId);
     this.showAssignedContainers(assignments);
+    document.querySelector("#edit-container-panel .panel-footer").hidden = !!userContextId;
```

Comment:
This seems to set `hidden=""` on the edit panel. Which, combined with `[hidden] { display: none !important }` seems to always prevent the footer from showing up ...

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/71928/27338134-00afbeae-559a-11e7-97db-0198c623dc4c.png">

I think this needs to be `if (!!userContextId) { document.querySelector("#edit-container-panel .panel-footer").removeAttribute('hidden'); }` ... because when I remove the hidden attribute completely, the footer shows up.